joomla/joomla-cms
# [5.3] CMSPlugin: Deprecate use of DispatcherAware and LanguageAware

### Summary of Changes

Deprecate use of DispatcherAware and LanguageAware within CMSPlugin class.

Plugin should use DispatcherAware it on its own.
And LanguageAware is a leftover from some changes in past, and does not used anywhere (I did not found).

### Testing Instructions

Code review.
Navigate around the site.
Edit site content. 
Use finder indexer.
All should work as before.


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/419

Linked:
- https://github.com/joomla/joomla-cms/pull/39387
